### PR TITLE
refactor: Introducing Suspense wrapper

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves []

### Description

This PR reduces the reusability of React.Suspense with ActivityIndicator

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

**Review by patrinoua — APPROVED** [review]
Looks great 🎈

File: src/Screens/_helpers/SuspenseWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Flex } from "palette"
```
Comment (pvinis) [inline src/Screens/_helpers/SuspenseWrapper.tsx:1]: whats with the `_helpers` naming? 🤔

File: src/Screens/HomeTabs/HomeTabs.tsx [hunk 2]
```diff
@@ -26,19 +24,19 @@ export const HomeTabs = () => {
   return (
     <TabsContainer header={Header}>
       <Tabs.Tab name="Artists" label="Artists">
-        <Suspense fallback={<TabsFlatList data={[0]} renderItem={() => <ActivityIndicator />} />}>
+        <SuspenseWrapper tabs>
```
Comment (pvinis) [inline src/Screens/HomeTabs/HomeTabs.tsx:27]: next thing we should do is move these inside the actual components, not here.

File: src/Screens/_helpers/SuspenseWrapper.tsx (hunk 1 above)
Comment (rajsam003) [reply src/Screens/_helpers/SuspenseWrapper.tsx:1]: It's in the Screens folder. I and @patrinoua  have decided to use `_` for helpers and shared folder names as it comes to the top and it's easier to navigate.

File: src/Screens/HomeTabs/HomeTabs.tsx (hunk 2 above)
Comment (rajsam003) [reply src/Screens/HomeTabs/HomeTabs.tsx:27]: Sure

**Review by pvinis — APPROVED** [review]

**Pull request merged**